helm-charts/azure-api-management-gateway/README.md

```diff
@@ -121,6 +121,13 @@ their default values.
 | `probes.liveness`                                      | Configuration for liveness probes of the container                                                                                                                                                                                                    | Uses `/status-0123456789abcdef` as endpoint for HTTP probes |
 | `securityContext`                                      | Privilege and access control settings for the  container                                                                                                                                                                                              | `{}`                                                        |
 | `podSecurityContext`                                   | Privilege and access control settings for the  pod                                                                                                                                                                                                    | `{}`                                                        |
+| `backup.enabled`                                   | If enabled will store a backup copy of the latests downloaded configuration on a storage volume                                                                                                                                                                                                 | `false`                                                        |
```

Comment:
Sorry for being a pain, but would you mind moving `backup.*` under `gateway.configuration`, for example `gateway.configuration.backup.enabled`?
```suggestion
| `backup.enabled`                                   | If enabled will store a backup copy of the latests downloaded configuration on a storage volume                                                                                                                                                                                                 | `false`                                                        |
```

This is to keep things close to the configuration and its endpoint given that's what we are backing up.